program_transformers/src/mpl_core_program/v1_asset.rs

```diff
@@ -536,3 +542,58 @@ fn convert_keys_to_snake_case(plugins_json: &mut Value) {
         _ => {}
     }
 }
+
+/// Updates the `asset_authority` for all assets that are part of a collection in a batch.
+/// This function performs a cursor-based paginated read and batch update.
+async fn update_group_asset_authorities<T: ConnectionTrait + TransactionTrait>(
+    conn: &T,
+    group_key: Vec<u8>,
+    authority: Vec<u8>,
+    slot: i64,
+) -> ProgramTransformerResult<()> {
+    let mut after = None;
+
+    let mut query = asset_grouping::Entity::find()
+        .select_only()
+        .column(asset_grouping::Column::AssetId)
+        .filter(asset_grouping::Column::GroupKey.eq(group_key.clone()))
+        .cursor_by(asset_grouping::Column::AssetId);
+    let mut query = query.first(1_000);
+
+    loop {
+        if let Some(after) = after.clone() {
+            query = query.after(after);
+        }
+
+        let entries = query.all(conn).await?;
```

Comment:
Because the query contains a `select_only`, I have found that it gets angry because `query.all()` tries to populate a whole `Model` but it only has the `AssetID` value.  

So I think we need to do something like:
```
#[derive(Debug, FromQueryResult, Clone)]
struct AssetIdQueryResult {
    pub asset_id: Vec<u8>,
}
...
let mut query = asset_grouping::Entity::find()
    .select_only()
    ...
    .into_model::<AssetIdQueryResult>();
```

which I just got from examples here: https://www.sea-ql.org/SeaORM/docs/advanced-query/custom-select/